bin/scheduler.py

```diff
@@ -24,9 +24,8 @@
 
 for algorithm, algo_config in services_config.iteritems():
     scheduler_options = algo_config['scheduler_options']
-    plugin = getattr(poll_tasks, scheduler_options['plugin'])
     CELERYBEAT_SCHEDULE[algorithm] = {
         'task': 'lib.app.task_runner',
         'schedule': timedelta(seconds=scheduler_options['interval_secs']),
-        'args': (plugin, scheduler_options['plugin_args'])
+        'args': (PollTask, {'plugin_name': scheduler_options['plugin']})
```

Comment:
so we cannot pass the plugin_args anymore ? not that we used them, but if so you can remove it from the README?